pkg/registry/registry.go

```diff
@@ -0,0 +1,17 @@
+package registry
+
+// ClusterStore stores mapping of clusters
+// and the shard they belong to
+type ClusterStore interface {
+	AddClusterToShard() error
+	AllAllClustersForShard() error
```

Comment:
Probably mean AddAllClustersForShard()? Also should we be consistent between *ToShard and *ForShard?